Phonebook.Frontend/src/app/services/api/current-user.service.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('CurrentUserService', () => {
       expect(user).toBe('username');
     });
 
```

Comment:
You have to change the mock response as well. 

As you are changing this. Please add a test for the other function (hasImage) aswell.